oneview_redfish_toolkit/app.py

```diff
@@ -62,3 +62,42 @@
 app.register_blueprint(manager)
 app.register_blueprint(odata)
 app.register_blueprint(thermal)
+
+if __name__ == "__main__":
```

Comment:
We don't need this if for main anymore...

Reply:
source of this please. Don't need also don't make it wrong, is a pattern long used.

Reply:
done